src/test/resources/MissedNotificationOutput.json

```diff
@@ -0,0 +1 @@
+["{\n    \"subscriptionName\" : \"Subscription_1\",\n    \"AggregatedObject\" : {\n    \"fileInformation\":[\n    {\n        \"extension\":\"jar\",\n        \"classifier\":\"\"\n    }\n    ],\n    \"buildCommand\":null,\n    \"testCaseExecutions\":[\n    {\n        \"testCaseFinishEventId\":\"11109351-41e0-474a-bc1c-f6e81e58a1c9\",\n        \"testCaseStartedTime\":1481875925916,\n        \"testCaseStartedEventId\":\"cb9d64b0-a6e9-4419-8b5d-a650c27c59ca\",\n        \"testCaseFinishedTime\":1481875935919,\n        \"testCase\":\n        {\n            \"conclusion\":\"SUCCESSFUL\",\n            \"verdict\":\"PASSED\",\n            \"tracker\":\"My Other Test Management System\",\n            \"id\":\"TC5\",\n            \"uri\":\"https://other-tm.company.com/testCase/TC5\"\n        }\n    }\n    ],\n    \"id\":\"6acc3c87-75e0-4b6d-88f5-b1a5d4e62b43\",\n    \"time\":1481875891763,\n    \"type\":\"ARTIFACT_1\",\n    \"gav\":\n    {\n        \"groupId\":\"com.mycompany.myproduct\",\n        \"artifactId\":\"sub-system\",\n        \"version\":\"1.1.0\"\n    }\n}\n}"]
```

Comment:
Same here. Remove Escape characters and "\n" characters, if possible. Indent JSON content.